src/main/scheduler/scheduler.c

```diff
@@ -104,6 +106,8 @@ static int16_t taskCount = 0;
 static uint32_t nextTimingCycles;
 #endif
 
+static timeUs_t lastFailsafeCheck = 0;
```

Comment:
either use `micros()` or some other type (int32_t, uint32_t)

Reply:
Should have been timeMs_t. I started off measuring in us, but then, realising the failsafe timings are all expressed in ms, switched to that.